# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: kinkie@squid-cache.org-20151109110536-tyi0em1kvypffv1x
# target_branch: ../trunk
# testament_sha1: 5edd7e7761be25eec7489c18e2c0278b1e21f418
# timestamp: 2015-11-09 16:38:04 +0100
# base_revision_id: squid3@treenet.co.nz-20151101121711-\
#   6xtmeycwsl00midx
# 
# Begin patch
=== renamed file 'src/cache_manager.cc' => 'src/CacheManager.cc'
=== modified file 'src/Makefile.am'
--- src/Makefile.am	2015-10-09 09:23:22 +0000
+++ src/Makefile.am	2015-11-09 11:05:36 +0000
@@ -236,7 +236,7 @@
 	cache_cf.cc \
 	CacheDigest.h \
 	CacheDigest.cc \
-	cache_manager.cc \
+	CacheManager.cc \
 	NeighborTypeDomainList.h \
 	CachePeer.cc \
 	CachePeer.h \
@@ -1013,7 +1013,7 @@
 	cache_cf.h \
 	YesNoNone.h \
 	tests/stub_cache_cf.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_comm.cc \
 	tests/stub_debug.cc \
 	tests/stub_errorpage.cc \
@@ -1167,7 +1167,7 @@
 	tests/stub_store_swapout.cc \
 	tools.h \
 	tests/stub_tools.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_UdsOp.cc \
 	tests/testACLMaxUserIP.cc \
 	tests/testACLMaxUserIP.h \
@@ -1265,7 +1265,7 @@
 	tests/stub_EventLoop.cc \
 	time.cc \
 	BodyPipe.cc \
-	cache_manager.cc \
+	CacheManager.cc \
 	cache_cf.h \
 	AuthReg.h \
 	YesNoNone.h \
@@ -1588,7 +1588,7 @@
 	cache_cf.h \
 	YesNoNone.h \
 	tests/stub_cache_cf.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_client_db.cc \
 	client_side_request.h \
 	tests/stub_client_side_request.cc \
@@ -1707,7 +1707,7 @@
 	cache_cf.cc \
 	CachePeer.cc \
 	CachePeer.h \
-	cache_manager.cc \
+	CacheManager.cc \
 	carp.h \
 	tests/stub_carp.cc \
 	cbdata.cc \
@@ -1945,7 +1945,7 @@
 	BodyPipe.cc \
 	CacheDigest.h \
 	tests/stub_CacheDigest.cc \
-	cache_manager.cc \
+	CacheManager.cc \
 	cache_cf.h \
 	AuthReg.h \
 	YesNoNone.h \
@@ -2195,7 +2195,7 @@
 	cache_cf.cc \
 	CachePeer.cc \
 	CachePeer.h \
-	cache_manager.cc \
+	CacheManager.cc \
 	CacheDigest.h \
 	tests/stub_CacheDigest.cc \
 	carp.h \
@@ -2463,7 +2463,7 @@
 	SBufAlgos.cc \
 	tests/stub_SBufDetailedStats.cc \
 	tests/stub_cache_cf.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_comm.cc \
 	tests/stub_cbdata.cc \
 	tests/stub_debug.cc \
@@ -2520,7 +2520,7 @@
 	tests/stub_EventLoop.cc \
 	time.cc \
 	BodyPipe.cc \
-	cache_manager.cc \
+	CacheManager.cc \
 	cache_cf.h \
 	AuthReg.h \
 	YesNoNone.h \
@@ -2867,7 +2867,7 @@
 	cache_cf.h \
 	YesNoNone.h \
 	tests/stub_cache_cf.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_client_side_request.cc \
 	tests/stub_comm.cc \
 	tests/stub_debug.cc \
@@ -2972,7 +2972,7 @@
 	cache_cf.h \
 	YesNoNone.h \
 	tests/stub_cache_cf.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_cbdata.cc \
 	tests/stub_debug.cc \
 	tests/stub_HelperChildConfig.cc \
@@ -3017,7 +3017,7 @@
 tests_testUfs_SOURCES = \
 	tests/testUfs.cc \
 	tests/testUfs.h \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_client_db.cc \
 	tests/stub_CollapsedForwarding.cc \
 	tests/stub_HelperChildConfig.cc \
@@ -3299,7 +3299,7 @@
 	YesNoNone.h \
 	tests/stub_cache_cf.cc \
 	client_db.h \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_client_db.cc \
 	tests/stub_client_side_request.cc \
 	tests/stub_debug.cc \
@@ -3382,7 +3382,7 @@
 	YesNoNone.cc \
 	RefreshPattern.h \
 	cache_cf.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	CacheDigest.h \
 	tests/stub_CacheDigest.cc \
 	CachePeer.cc \
@@ -3636,7 +3636,7 @@
 	tests/stub_HelperChildConfig.cc \
 	tests/stub_libmem.cc \
 	tests/stub_cache_cf.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_cbdata.cc \
 	tests/stub_store.cc \
 	tests/stub_store_stats.cc \
@@ -3681,7 +3681,7 @@
 	tests/stub_HelperChildConfig.cc \
 	tests/stub_libmem.cc \
 	tests/stub_cache_cf.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_store.cc \
 	tests/stub_stmem.cc \
 	tests/stub_store_stats.cc \
@@ -3720,7 +3720,7 @@
 	cache_cf.h \
 	YesNoNone.h \
 	tests/stub_cache_cf.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_cbdata.cc \
 	tests/stub_debug.cc \
 	tests/stub_libmem.cc \
@@ -3756,7 +3756,7 @@
 	StatHist.cc \
 	StatHist.h \
 	String.cc \
-	tests/stub_cache_manager.cc \
+	tests/stub_CacheManager.cc \
 	tests/stub_comm.cc \
 	tests/stub_debug.cc \
 	tests/stub_DelayId.cc \

=== renamed file 'src/tests/stub_cache_manager.cc' => 'src/tests/stub_CacheManager.cc'
# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWcsUwoMAA4RfgAAwUGP/91sC
EAS////wUAYe4UQ4Naptdiq21Vm0JKTJMUw02lPCJ6DU8UeSMmgaAShNRhNMiepqmmnqekGgAADQ
JEQU9Iyap5RgR6hppoNAHqAMAAAAANBoA0A0AkimRoCYBU8JhTU/Qo8nqhk009RkgqSCxrNebMhm
Qm8YfonMxkaa3z2ARYGYEbfL5sYeI6Dfes2MUrNsqvGWyIvXgu9VTBNg2A39fFJHOlnh7WpcMmYW
HRrOkTMLe7Ui2cyiGQ7TFB2ekwTLo5u4LbJh5cDNIPgH0/vjwXNoVs92973+KmnAfUvnby96KNJc
r0+zeAnI/Un3PkGEcDmY3JzRCyOPl5RvL9ajeyRtxXBIVIcglC6V154op6EJ68XdBq2zRnFlGQw4
VhQwKuZpMLsYTx4c1xKLDsAfkgLh2mw3FMzZM1obMOY5FUiChoVMuXwk56Z4M9pzJgRpHkNSyIqD
fUQ66mxvrNUgo/ccL8tHsgbh3lQYUrQ0jMtKA8NGAjAYR9DrnRUqlcIYRYUpmrMBqaqyplAnw1hL
N7SI0hzZDmFRh+bbRBBaFMx8OWER047Qhl2muJzaRsFJepxyvXbYI9xHmf8IlZWnUuO7FgYUQLoM
nadkSaZDkTERh7m5sZxMkSKo/WBh6FtCw9TRhTUa50Y4nsGgkdiBImcu8lkO2wb+BkSmalZORgPJ
jqQCgu6rkOaaIhHxxDEWBVY1F5PuLfuDMzs6lbQdTYZNRVJMyEGGbu2nC3OMYCupyNYlisiBnKR4
RcwbgqdT44DtOt6ZO923Gi29b1NYEuzB6AlRsHUQfhzcw9dt7FxzOoxAYrYoQ09aEIz7cxScXGBW
G4NxYZxNqCFLcCVExQPIrxdF2dTDxOhwSdLtJIFdpKt6Mg5mxg4mZUyUzccxx0WmPPbnzQcpNzOc
DkTYlphuGvhgzPkQxgRkqB51LHmQJ1QdM9S5ShgwWiOoOdhaXfHnYnSQpf/VbIrNhnFsOTgKIa7M
QFOlSVSDkjI4G3OVZjjlYDUzXLbfWDWbzh3yd1lAUhDpaKhcLMVbvxCyXtgh9G7pwmY4/2v4xiCx
BlWmGG/PyfdmBmaDD0K+FIEiJSDuAJ0eo20j5/GRciAOm64pOmCF2EhiMbuBJqs0lo3XqFMkL9sO
XROChxFIGYYadel6CC0PXzxuHGvmIgDmBkceoJaiRcMYjndc5/c+hcVTIXU695VoFfbv6Dcd4CLW
flIL4fCnnQs6HDFsigqROigeIGpE4gzndGKzkK5CYqzrgYmUztPrLVx4PG1DTFTJ4Jqo8nfMSM1g
Jxft7mfsT9PQvf42b1R5N9k8N1jV/hyQiX+aHcHVMQ9IW2um/e/ZtIRKcLc4mpOC81V1jdj6nU6H
MkQOB7elRX2KpsisKzIMxH6VK9V7rZjMwI6ELMQDN2g+oBgIt9Ypkkj2+AjsNKDOS+iAs5pQQbli
Ck4hfQCaUwi1sBfldLqhXisuJlp7ct/mNyWoLUVvg7Aye/sItXIGFc4WgtrdE/N7nKIRmLfCyXXM
jcpAfLYmZLZNLFEGZxzYEEhR5pyIvTfPOEMmBQyglMyGXgO27ki8wwbELJxmSjQo13NWmTImnYmI
aFUOAh8h9WR5e3EQbzYSWxXYL5+O8jcRzC2tA3nmEUKst6TKT5QoFEltkwtrLrtE16VddDP8hWR6
zBAmJdyFLc+T11CIX3Fo8X5oXEXhs4d24wrZC79y3g5EooET8iUhAKBGJVw799xGIK+wgtMNDBDW
kqpFuPsI7Vz5q9BYmkl4Wa7Ej+Yr3MRGwiPkaDwgeEf+LuSKcKEhlimFBg==
